src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import cn from 'classnames';
+
+import { SortFields } from '../../types/sortFields';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setSelectedField: (string: SortFields) => void;
+  selectedField: string;
+  todos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({
+  setSelectedField,
+  selectedField,
+  todos,
+}) => {
+  const isAnyCompleted = todos.every(todo => !todo.completed);
```

Comment:
It is probably not a perfect naming here, but it is not critical

Better: `hasCompletedTodos`, etc

Reply:
And a slightly better way is :

```suggestion
  const hasCompletedTodos = !!todos.find(todo => todo.completed);
```

Your version needs to check each todo, while the other just searches for the first completed todo, so potentially it would not need to iterate through all todos